votepy/testing/rules/chamberlin_courant.py

```diff
@@ -0,0 +1,90 @@
+from votepy.ordinal_election import OrdinalElection
+
+from votepy.testing.structure.structure import rule, get_implementation, impl
+from votepy.testing.algorithms.base_algorithm import BaseAlgorithm
+from votepy.testing.algorithms.generic_brute_force import BruteForce
+from votepy.testing.algorithms.generic_greedy import Greedy
+
+from typing import Union, Iterable
+
+
+@rule()
+def chamberlin_courant(voting: Union[OrdinalElection, list[int]], size_of_committee: int, number_of_scored_candidates: int, algorithm: BaseAlgorithm) -> list[int]:
+    """Chamberlin-courant rule
+    Args:
+        `voting` (`Union[OrdinalElection, list[int]]`): Voting for which the function calculates the committee
+        `size_of_committee` (`int`): Size of the committee
+        `number_of_scored_candidates` (`int`): Number of scored candidates using k-borda rule
+
+    Returns:
+        `list[int]`: List of chosen candidates
+    """
+    implementation = get_implementation(chamberlin_courant, algorithm)
+    implementation(voting, size_of_committee,
+                   number_of_scored_candidates, algorithm)
+
+
+@impl(rule=chamberlin_courant, algorithm=BruteForce)
+def chamberlin_courant_brute_force(voting, size_of_committee, number_of_scored_candidates, brute_force: BruteForce = BruteForce()) -> list[int]:
+    def scoring_function(committee: Iterable[int], voting: OrdinalElection):
+        committee = set(committee)
+        score = 0
+        for vote in voting:
+            for i, candidate in enumerate(vote):
+                if candidate in committee:
+                    score += max(number_of_scored_candidates - i, 0)
+                    break
+        return score
+
+    brute_force.prepare(scoring_function)
+    return brute_force.solve(voting, size_of_committee)
+
+
+@impl(chamberlin_courant, Greedy)
```

Comment:
I think we should be consistent here (with i.e. line 27), either use keywords everywhere, or nowhere

Reply:
I think this version without keywords is self-explanatory enough, I will change line 27.